api-inference-community/docker_images/pyannote/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM tiangolo/uvicorn-gunicorn:python3.8
+LABEL maintainer="me <me@example.com>"
+
+# Add any system dependency here
+# RUN apt-get update -y && apt-get install libXXX -y
+
+COPY ./requirements.txt /app
+RUN pip install --no-cache-dir -r requirements.txt
+COPY ./prestart.sh /app/
+
+
+# Most DL models are quite large in terms of memory, using workers is a HUGE
+# slowdown because of the fork and GIL with python.
+# Using multiple pods seems like a better default strategy.
+# Feel free to override if it does not make sense for your library.
+ARG max_workers=1
+ENV MAX_WORKERS=$max_workers
+ENV HUGGINGFACE_HUB_CACHE=/data
```

Comment:
Can you confirm that the cache env key is using this ? Or does PYANNOTE defines another variable to locate the cache directory ? 

(This is important for the API to load the pods faster, by sharing a disk for the model files)

Reply:
`pyannote` uses `PYANNOTE_CACHE`.

Reply:
Ok, do you mind adding

```
ENV PYANNOTE_CACHE=/data
```

Reply:
> Btw @J-Petiot , `make style` should fix the quality thing.

Ok thanks it should be better.